[DS-196]: [Pyxis SCSS] Improve Readme (#174)
primait/pyxis: packages/pyxis-scss/README.md

- Typography
- and some useful functions.

### Components module
Here is where the things get complex. Pyxis gives you a powerful UI kit in order to get things done easily.

Each of these modules is paired with a [React](https://github.com/primait/pyxis/tree/master/packages/pyxis-react) and an [Elm](https://github.com/primait/pyxis/tree/master/packages/pyxis-elm) version of the component.

### Atoms module
This module contains the atomic classes that we expose for all the modules contained in the foundations. For example, it contains classes like `color-brand-base` or `padding-m`.

**Please note:**
Pyxis is **NOT** a utility-first CSS framework (like Tailwind), it is advisable to use these classes only when they can avoid writing a custom class, in general it is better not to use more than 2-3 at the same time, in that case it is better to create a custom class.

---

> ## ⚙️ Install Pyxis SCSS with Pyxis React or Pyxis Elm

Reviewer: ```suggestion
## ⚙️ Usage with React or Elm
```

Using Pyxis SCSS paired with components developed in _Pyxis Elm_ or _Pyxis React_ is the most recommended way.
To do so, all you need to do is to install the React or Elm package and be sure to have the Pyxis SCSS package too, otherwise add it manually.

For more information check the specific package:

- [Use Pyxis with React](https://github.com/primait/pyxis/tree/master/packages/pyxis-react)
- [Use Pyxis with Elm](https://github.com/primait/pyxis/tree/master/packages/pyxis-elm)

---

> ## ⚙️ Install Pyxis SCSS standalone version

Reviewer: ```suggestion
## ⚙️ Standalone usage
```

Pyxis can be used as a standalone unit by projects which do not rely on _Elm_ or _React_ for the frontend.

> ### Install Pyxis SCSS

Reviewer: ```suggestion
### Npm module usage
```

> To install `@pyxis/scss` you will need to follow the instructions for installing through [JFrog Artifactory](https://www.jfrog.com/confluence/display/RTF/Npm+Registry) described in the [Setup](https://github.com/primait/pyxis#-setup) chapter of the monorepo.

Reviewer: ```suggestion
**Prerequisite**: [had configured Jfrog Artifactory cli](https://github.com/primait/pyxis#-setup) chapter of the monorepo.

To install `@pyxis/scss` via _npm_ simply run: 
```sh
$ npm i @pyxis/scss
```
```

### Configuring Pyxis (optional)
You can redefine some variables (those defined in `@pyxis/scss/src/scss/config.scss`) by writing something like that:

```scss
// Require the configuration module and override the $fontDisplay variable.
// Using the @use directive we ensure CSS is never repeated in case of multiple requirements.
@use "~@pyxis/scss/src/scss/config.scss" with ($fontDisplay: "fallback");

// Pyxis module will now use your $fontPath variable.
@forward "~@pyxis/scss/src/scss/pyxis.scss";
```

---

## 🪄 Use Pyxis SCSS

> As seen in the previous section `@pyxis/scss` is built with the sass modules, we can use them in several ways.

Reviewer: Qui proverei a spiegare un po' di più nel dettaglio cosa significa "usare tutto pyxis", e in quali casi conviene utilizzare solo `base & foundations` oppure `components & atoms`.

Aggiungerei anche una sezione "usare Pyxis CSS" cioè la versione full precompilata.

Reviewer 2: > Qui proverei a spiegare un po' di più nel dettaglio cosa significa "usare tutto pyxis", e in quali casi conviene utilizzare solo base & foundations oppure components & atoms.

@alessiosantangelo queste cose sono già scritte sotto in ogni capitolo, non sarebbe una ripetizione?

Ok invece per aggiungere la sezione "usare Pyxis CSS" anche se scriverei che è comunque sconsigliato

Author: Anche a me sembra già spiegato bene nei capitoli sottostanti, ok invece per Pyxis CSS
